app/src/main.ts

```diff
@@ -168,6 +169,52 @@ if (shouldQuit) {
           });
         });
       });
+
+      if (isOSX() && appArgs.accessibilityPrompt) {
+        const mediaKeys = [
+          'MediaPlayPause',
+          'MediaNextTrack',
+          'MediaPreviousTrack',
+          'MediaStop',
+        ];
+        const globalShortcutsKeys = appArgs.globalShortcuts.map((g) => g.key);
+        const mediaKeyWasSet =
+          globalShortcutsKeys.filter((g) => mediaKeys.includes(g)).length > 0;
```

Comment:
Nit: no need to look at the whole array, let's stop at the first match:

```suggestion
        const mediaKeyWasSet =
          globalShortcutsKeys.find((g) => mediaKeys.includes(g));
```